### PR TITLE
Update contributing.md

### PR DESCRIPTION
**Pull request by puneetse**
<s>Update header to use "Clear Linux* OS" per TCS recommendation.</s>

Change header to "Reporting Security Concerns"

### CODE REVIEW

**Comment by fenrus75** [comment]
no *

**Comment by ahkok** [comment]
Let's not mix changes like this - split them into 2 separate PRs the next time. The second change is OK of course.

File: contributing.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 
-## Clear Linux OS - Software Contribution Guide
+## Clear Linux* OS - Software Contribution Guide
```
Comment (ahkok) [inline contributing.md]: please drop this line from this PR.

Comment (puneetse) [reply contributing.md]: Done.

**Pull request merged**